Automattic/jetpack
# Story Block: allow multiple stories per post

In https://github.com/Automattic/jetpack/pull/17342 stories were assigned the id of the post so that when displaying the same story several times on the page it resumes the progress where it was left off.
This should actually be considered a bug not a feature as it breaks having multiple stories in the same post.
At first we excluded this as the mobile app was only going to generate 1 story posts, but seeing [the announcement post](https://wordpress.com/blog/2021/03/08/introducing-wordpress-stories-a-new-way-to-engage-your-audience/) showcasing the bug now, I think it's best to fix this asap.

#### Changes proposed in this Pull Request:
Let's not use the post id to identify a story.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Try adding 2 stories to a post
* View the post
* Try playing the 2 stories separately, it should work as expected
